src/category_theory/limits/over.lean

```diff
@@ -84,6 +86,119 @@ instance forget_preserves_colimits [has_colimits.{v} C] {X : C} :
   { preserves_colimit := λ F, by exactI
     preserves_colimit_of_preserves_colimit_cocone (colimit.is_colimit F) (forget_colimit_is_colimit F) } }
 
+/-- Given the appropriate pullback in C, construct a product in the over category -/
+def over_product_of_pullbacks (B : C) (F : discrete walking_pair ⥤ over B)
+[q : has_limit (cospan (F.obj walking_pair.left).hom (F.obj walking_pair.right).hom)]
+: has_limit F :=
```

Comment:
The colon should go to the end of the previous line (with a space before it. The goal is indented with two spaces.